lib/OpenQA/Command.pm

```diff
@@ -119,6 +119,7 @@ sub url_for ($self, $path) {
 }
 
 sub retry_tx ($self, $client, $tx, $retries = undef, $delay = undef) {
+    local $ENV{MOJO_CONNECT_TIMEOUT} //= 30;
```

Comment:
That shouldn't work. It's a `Mojo::UserAgent` attribute with lazy initialization.

Reply:
oh, right :)
fixed